src/App.tsx

```diff
@@ -1,25 +1,22 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { BrowserRouter, Routes, Route, Navigate } from "react-router-dom";
+
+import Header from "./Header";
+import ProductsPage from "./pages/ProductsPage";
+import SomePage2 from "./pages/SomePage2";
+import "./App.css";
 
 function App() {
```

Comment:
всегда лучше использовать стрелочные функции, занимают меньше места и удобнее
```suggestion
const App = () => {
```

Reply:
переделал